src/style.css

```diff
@@ -1,3 +1,65 @@
-body {
+:root {
+  --blue-color: #00acdc; /* Определение переменной для синего цвета */
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-align: left;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+}
+
+.nav_list li {
+  display: block;
+  margin-left: 20px;
+  position: relative;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: #000;
+  padding: 23 0;
```

Comment:
The padding value in the `.nav-link` class seems to be missing units. It should be specified in pixels (px), ems (em), or other valid CSS units.